mods_cocina_mappings/mods_to_cocina_name.txt

```diff
@@ -116,11 +116,8 @@ Names
       "type": "person",
       "identifier": [
         {
-          "value": "https://www.wikidata.org/wiki/Q7704207",
-          "type": "URI",
-          "source": {
-            "code": "wikidata"
-          }
+          "uri": "https://www.wikidata.org/wiki/Q7704207",
+          "type": "Wikidata"

```

Comment:
How do we decide casing? (e.g., lccn to LCCN vs. Wikidata).

Reply:
See also https://github.com/sul-dlss/dor-services-app/issues/1553 for inconsistent casing.

Reply:
See https://docs.google.com/spreadsheets/d/1d5PokzgXqNykvQeckG2ND43B6i9_CsjfIVwS_IsphS8/edit#gid=0&range=A131 for mapping MODS 3.7 term to COCINA term (the identifier value types section). The MODS/Standard Identifier Scheme type code should always be lowercase and may be downcased if needed.

I can put together a MODS>COCINA identifier type mapping doc if you let me know what format you need it in.